crl-janitor/sa.tf

```diff
@@ -14,3 +15,20 @@ resource "google_project_iam_member" "app_roles" {
   role     = local.sa_roles[count.index]
   member   = "serviceAccount:${google_service_account.app[0].email}"
 }
+
+resource "google_service_account" "client" {
+  count = var.enable ? 1 : 0
+
+  provider     = google.target
+  account_id   = "crl-janitor-client"
+  display_name = "crl-janitor-client"
+}
+
+resource "google_project_iam_member" "client_roles" {
+  count = var.enable ? length(local.client_sa_roles) : 0
+
+  provider = google.target
+  project  = var.google_project
+  role     = local.client_sa_roles[count.index]
+  member   = "serviceAccount:${google_service_account.client[0].email}"
+}
```

Comment:
if we don't need roles yet, maybe don't define this or the client_sa_roles? It doesn't seem likely at the moment that it will ever needs roles, so let's not write any code yet to give it roles.

Reply:
Without the role, I got complains from IntelliJ saying "Missing required properties: role "

Reply:
Sorry, I meant remove the entire `resource "google_project_iam_member" "client_roles"`.

Reply:
done